### ISSUE TITLE
I can not catch pokemon

### ISSUE THREAD

**Issue by hitman200** [finndev/PokeBuddy#950]
what happened to me I can not catch pokemon but can spin pokestop there is a limit to how many you can catch

**Comment by futi666** [comment]
i cannot spin pokestops but catch pokemons...

**Comment by etowery22** [comment]
No, if you can't spin pokestops and pokemon flee from you, then you have a softban. If pokestops stop working and pokemon do not show up, then you have an IP ban.

**Comment by hitman200** [comment]
yes but I can spin but not catch

**Comment by 404unknown** [comment]
#912

**Comment by JewBagel** [comment]
lmao thats not much help half are saying permabans are a thing the other half are saying its server problems and people that arent botting are having the same problems

**Comment by 404unknown** [comment]
@JewBagel  See the link? Im replying to someone that said "i cannot spin pokestops but catch pokemons..."

**Comment by JewBagel** [comment]
yeah i see the link im just saying people are so confused on there
